app/components/file-browser/component.js

```diff
@@ -199,7 +199,7 @@ export default Ember.Component.extend(Analytics, {
             this.get('toast').error(response.message_long || response.message || response);
         },
         success(_, __, file, response) {
-            this.send('track', 'upload', 'track', 'Quick Files - Upload');
+            this.get('analytics').track('upload', 'track', 'Quick Files - Upload');
```

Comment:
Should be `this.get('analytics').track('file', 'upload', 'Quick Files - Upload');`